.github/workflows/unused-styles.yml

```diff
@@ -0,0 +1,30 @@
+name: Check unused styles
+
+on:
+  workflow_call:
+  pull_request:
+    types: [opened, synchronize]
+    branches-ignore: [staging, production]
+    paths: ['src/styles/**', 'src/components/**', 'src/pages/**', '**.tsx', '**.ts']
```

Comment:
There's no sense in including these other paths if you're including `**.tsx` and `**.ts`. We should remove those last two imo. We'd also want to include `src/hooks` and `src/libs/Navigation`, which has some UI code.

TBH though, this check only takes ~12 seconds. So I'd actually recommend just doing this:

```
paths: ['src', '.github/workflows/unused-styles.yml', '.github/scripts/findUnusedStyles.ts']
```

It's pretty broad, but should check everything and still skip things that are unrelated like HelpDot or GitHub Actions PRs.